src/form/Checkbox/CheckboxTheme.ts

```diff
@@ -43,57 +54,116 @@ export interface CheckboxTheme {
 }
 
 const baseTheme: Partial<CheckboxTheme> = {
-  base: 'inline-flex items-center w-full',
+  base: 'inline-flex items-center w-full group',
   label: {
-    base: 'ml-2.5 w-full',
+    base: 'text-surface-content dark:text-waterloo light:text-charade ml-2.5 w-full',
+    checked: 'checked dark:text-athens-gray light:text-vulcan',
```

Comment:
@steppy452 - should these be referecing colors like this?

Reply:
will replace normal one